.github/workflows/ci.yml

```diff
@@ -18,6 +18,11 @@ jobs:
           sudo apt-get install libhdf5-serial-dev zlib1g-dev libatlas-base-dev lcov swig3.0 libpcsclite-dev
           mkdir -p ~/bin/ && ln -s /usr/bin/swig3.0 ~/bin/swig && export PATH=~/bin/:$PATH
           pip install -e .[ci,test,yubikey] # Install with CI deps
+      - name: setup github user
+        shell: bash
+        run: |
+          git config --global user.name oll-bot
+          git config --global user.email developers@openlawlib.org
```

Comment:
Is oll-bot necessary because of the testing framework?

Reply:
Yes, because the testing framework needs to create commits and that operation fails if user's data was not specified